lms/djangoapps/courseware/models.py

```diff
@@ -29,10 +30,37 @@
 log = logging.getLogger("edx.courseware")
 
 
+def chunks(items, chunk_size):
+    """
+    Yields the values from items in chunks of size chunk_size
+    """
+    items = list(items)
+    return (items[i:i + chunk_size] for i in xrange(0, len(items), chunk_size))
+
+
+class ChunkingManager(models.Manager):
```

Comment:
We actually don't chunk the queries anymore; we just make one big one: https://github.com/edx/edx-platform/blob/master/lms/djangoapps/instructor_task/subtasks.py#L105